code/network/multimsgs.cpp

```diff
@@ -2173,76 +2195,98 @@ void process_netgame_update_packet( ubyte *data, header *hinfo )
 
 */
 // send a request or a reply for mission description, if code == 0, request, if code == 1, reply
+// Cyborg17 - Now recursive for longer descriptions!
 void send_netgame_descript_packet(net_addr *addr, int code)
 {
-	ubyte data[MAX_PACKET_SIZE],val;
-	int desc_len;
+	ubyte data[MAX_PACKET_SIZE];
 	int packet_size = 0;
 
-	// build the header
-	BUILD_HEADER(UPDATE_DESCRIPT);
+	// Get this out of the way, because we don't want to waste our time if the addr is bad.
+	Assert(addr != nullptr);
+	if (addr == nullptr) {
```

Comment:
Double check the use of Assert, I forget if it allows resumption or not.

Reply:
I'm not sure. This is a leftover from the original code and should probably be removed/changed though. I'll add it to the list.